interfaces/Libraries.ts

```diff
@@ -116,11 +116,35 @@ from pyannote.core import Segment
 excerpt = Segment(start=2.0, end=5.0)
 model.crop("file.wav", excerpt)`;
 
-const tensorflowtts = (model: ModelData) =>
+const tensorflowttsTextToMel = (model: ModelData) =>
+`from tensorflow_tts.inference import AutoProcessor, TFAutoModel
+
+processor = AutoProcessor.from_pretrained("${model.modelId}")
+model = TFAutoModel.from_pretrained("${model.modelId}")
+`;
+
+const tensorflowttsMelToWav = (model: ModelData) =>
 `from tensorflow_tts.inference import TFAutoModel
+
 model = TFAutoModel.from_pretrained("${model.modelId}")
+audios = model.inference(mels)
 `;
 
+const tensorflowttsUnknown = (model: ModelData) =>
+`from tensorflow_tts.inference import TFAutoModel
+
+model = TFAutoModel.from_pretrained("${model.modelId}")
+`;
+
+const tensorflowtts = (model: ModelData) => {
+	if (model.tags.includes("text-to-mel")){
+		return tensorflowttsTextToMel(model);
+	} else if (model.tags.includes("mel-to-wav")) {
+		return tensorflowttsMelToWav(model);
+	}
+	return tensorflowttsUnknown(model);
```

Comment:
Nitpick but this can probably be reduced to return `tensorflowttsTextToMel` or `tensorflowttsUnknown`, without needing to specify `tensorflowttsMelToWav` as it's the same as `tensorflowttsUnknown`

Reply:
`tensorflowttsMelToWav` has inference which `tensorflowttsUnknown` does not (the function signature is different for other models), so they are a bit different